SAMtags.tex

```diff
@@ -90,11 +92,13 @@ \section{Standard tags}
   {\tt PQ} & i & Phred likelihood of the template \\
   {\tt PT} & Z & Read annotations for parts of the padded read sequence \\
   {\tt PU} & Z & Platform unit \\
-  {\tt QT} & Z & Barcode ({\tt BC} or {\tt RT}) phred-scaled base qualities \\
-  {\tt Q2} & Z & Phred quality of the mate/next segment sequence in the {\tt R2} tag \\
+  {\tt QT} & Z & Phred quality of the sample-barcode sequence in the {\tt BC} (or {\tt RT}) tag. Same encoding as {\sf QUAL}. \\
+  {\tt QX} & Z & Phred quality of the molecular-barcode sequence in the {\tt RX} tag. Same encoding as {\sf QUAL}. \\
+  {\tt Q2} & Z & Phred quality of the mate/next segment sequence in the {\tt R2} tag. Same encoding as {\sf QUAL}.\\
   {\tt R2} & Z & Sequence of the mate/next segment in the template \\
   {\tt RG} & Z & Read group \\
   {\tt RT} & Z & Barcode sequence (deprecated; use {\tt BC} instead) \\
+  {\tt RX} & Z & Raw (uncorrected) molecular-barcode bases, with any quality scores stored in the {\tt QX} tag. \\
```

Comment:
I think we should add verbiage here about what to do if you have multiple molecular barcode reads.  In practice I believe most (or at least > 1) or us are concatenating multiple molecular barcode reads with hyphen separators.